functests/utils/pods/pods.go

```diff
@@ -223,3 +224,27 @@ func GetPerformanceOperatorPod() (*corev1.Pod, error) {
 
 	return &pods.Items[0], nil
 }
+
+//GetCpusNumber returns the cpus number that are used by mco pod
+func GetCpusNumber(pod *corev1.Pod) (int, error) {
+	cmdGetCPU := []string{"bash", "-c", fmt.Sprintf("cat /sys/fs/cgroup/cpuset/cpuset.cpus")}
+	stdout, err := ExecCommandOnPod(testclient.K8sClient, pod, cmdGetCPU)
+	if err != nil {
+		return -1, fmt.Errorf("unable to get cpus number of the pod : %w", err)
+	}
+	if strings.Contains(string(stdout), "-") {
```

Comment:
best to use cpuset.Parse, see examples in functests/1_performance/cpu_management.go.